status-im/status-react
# Added ERC20 listing support

addresses #2468

### Summary:

Added support for ERC20 token listing in main wallet screen.

### Steps to test:
- Open Status
- Navigate wallet main screen: a line showing STT should appear (with eventually a non-0 amount)
- Step 3, etc.


status: ready
